lib/accesslint/ci/cli.rb

```diff
@@ -0,0 +1,17 @@
+require "thor"
+
+module Accesslint
+  module Ci
+    class CLI < Thor
```

Comment:
As much as I hate CamelCasingThingsThatAreActuallyCapitalized, I think the first fits with our [guides](https://github.com/thoughtbot/guides/blob/5deeaaa2006a7f895f697cc99668eadaa4abe693/style/ruby/README.md#5).